Bio/Graphics/GenomeDiagram/_CircularDrawer.py

```diff
@@ -1325,8 +1325,9 @@ def _draw_sigil_jaggy(self, bottom, center, top,
             headangle = endangle
             tailangle = min(startangle + min(height * tail_length_ratio / (center * teeth), angle), endangle)
 
-        assert startangle <= tailangle <= headangle <= endangle, \
-            (startangle, tailangle, headangle, endangle, angle)
+        if not startangle <= tailangle <= headangle <= endangle:
+            raise RuntimeError(startangle, tailangle,
```

Comment:
Does this give a sensible error message? It would probably be better to build an explicit message string here.